### PR TITLE
Remove some user-facing WDL specificity.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 import akka.actor._
-import cats.data.Validated.{Invalid, Valid}
 import cats.data.NonEmptyList
-import cats.syntax.validated._
+import cats.data.Validated.{Invalid, Valid}
 import cats.syntax.cartesian._
+import cats.syntax.validated._
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.core.{WorkflowId, WorkflowOptions, WorkflowOptionsJson, WorkflowSourceFilesCollection}
+import cromwell.core.{WorkflowId, WorkflowOptions, WorkflowOptionsJson, WorkflowSourceFilesCollection, _}
```
Comment (danbills) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: ToL can remove specific imports w/ the `_` present

**Comment by danbills** [comment]
👍

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala (hunk 1 above)
Comment (mcovarr) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: Yeah, it's annoying IntelliJ doesn't clean that up for us automatically.  😠

**Comment by ruchim** [comment]
:+1: bye bye WDL

**Pull request merged**